src/main/java/at/helpch/placeholderapi/expansion/server/util/VersionHelper.java

```diff
@@ -0,0 +1,99 @@
+/**
+ * MIT License
+ * <p>
+ * Copyright (c) 2021 TriumphTeam
+ * <p>
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ * <p>
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ * <p>
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package at.helpch.placeholderapi.expansion.server.util;
+
+import com.google.common.primitives.Ints;
+import org.bukkit.Bukkit;
+import org.bukkit.OfflinePlayer;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * Class for detecting server version for legacy support :(
+ */
+public final class VersionHelper {
+
+    public static final String NMS_VERSION = getNmsVersion();
+
+    private static final int V1_17 = 1_17_0;
+
+    private static final int CURRENT_VERSION = getCurrentVersion();
+
+    private static final boolean IS_PAPER = checkPaper();
+
+    public static final boolean IS_1_17_OR_HIGHER = CURRENT_VERSION >= V1_17;
+
+    /**
+     * Check if the server has access to the Paper API
+     * Taken from <a href="https://github.com/PaperMC/PaperLib">PaperLib</a>
+     *
+     * @return True if on Paper server (or forks), false anything else
+     */
+    private static boolean checkPaper() {
+        try {
+            Class.forName("com.destroystokyo.paper.PaperConfig");
+            return true;
+        } catch (ClassNotFoundException ignored) {
+            return false;
+        }
+    }
```

Comment:
I think this should be updated to work with the new Config location. Just in case this old path may eventually not exist anymore in paper.